### ISSUE TITLE
Have the possibility to locally reorder sounds - or session pre setup 

### ISSUE THREAD

**Issue by GuillaumeGwigwi** [benmiaou/chatluthier#9]
This might be managed by doing a "session" tab which is kind of a preset that only selects some sounds pre selection

**Comment by GuillaumeGwigwi** [comment]
Let's say I want to do a setup for a specific moment in the campaign, and i need some sound effects, I can go on a menu and select "Guillaume session X", this filters the available sounds with my preset.

The issue i am trying to answer here is:
- How do we manage to have an easy to use app even if we have a lot of sound, ambiance and/or theme background sound
-

**Comment by benmiaou** [comment]
A setting page with sections for 'Background', 'Ambient', and 'Soundboard' audio files, each with checkboxes to select or deselect which files are displayed on the main page. and you can save your preset for later. 
Good for me

**Comment by GuillaumeGwigwi** [comment]
Yes something like that, for now as we dont have any user management it will be available for everyone but i think it suits our needs

**Comment by benmiaou** [comment]
maybe use a session token to save the preset user side

**Comment by GuillaumeGwigwi** [comment]
Yeah it might be a solution but it might at some point imply that we lose preset if for any reason, the session is reseted
